modules/docs/src/main/scala/docs/AwsServiceList.scala

```diff
@@ -11,7 +11,11 @@ object AwsServiceList {
     val summary = getSummary()
     val supportedProtocols = Set(
       AwsJson1_0.id.name,
-      AwsJson1_1.id.name
+      AwsJson1_1.id.name,
+      RestJson1.id.name,
+      RestXml.id.name,
+      AwsQuery.id.name,
+      Ec2Query.id.name
     )
     val (supported, unsupported) =
```

Comment:
thought: we have this list already in `AwsProtocol.supportedProtocols`

Reply:
[f9221b1](https://github.com/disneystreaming/smithy4s/pull/1212/commits/f9221b1e6a4f73bf0065dd326156b742c90030e3)